src/index.ts

```diff
@@ -48,6 +50,19 @@ async function bootstrap() {
       debug: NODE_ENV === "development",
     }
 
+    // Setup Sentry
+    if (enableSentry) {
+      Sentry.init({
+        dsn: "https://caca4ef0651e4adbbd1f099702dbb779@sentry.io/1405985",
```

Comment:
@xtina-starr we should use the env variable for this correct?

Reply:
oh shoot I actually set it up that way and for got to remove this

Reply:
Good catch @l2succes! This was for the staging environment. I'll go ahead and generate a new key.